web/src/beta/lib/core/StoryPanel/hooks.ts

```diff
@@ -61,6 +63,19 @@ export default (
     [selectedPageId, isEditable],
   );
 
+  const handlePageTime = useCallback(
+    (page: StoryPage) => {
+      const timePointField = page.property?.timePoint;
+      const currentTime = timePointField?.timePoint?.value;
+      if (!currentTime) onTimeChange?.(new Date());
+      else {
+        const getNewDate = new Date(currentTime.substring(0, 19)).getTime();
+        return onTimeChange?.(new Date(getNewDate));
+      }
+    },
+    [onTimeChange],
+  );
```

Comment:
If I understand the logic here correctly, are we changing the time to the actual time/date on page change if there is no `timePoint?.value`? Hmmm maybe something to discuss with @lavalse